modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SearchResultResourceTest.java

```diff
@@ -427,11 +482,37 @@ private void _testPostSearchPageWithUserFacetConfiguration()
 	}
 
 	private void _testPostSearchPageZeroResults() throws Exception {
-		Page<SearchResult> page = _postSearchPage("shouldnotmatchanything");
+		SearchPage<SearchResult> page = _postSearchPage(
+			"shouldnotmatchanything");
 
 		Assert.assertEquals(0L, page.getTotalCount());
 	}
 
+	private SearchPage<SearchResult> _toSearchPage(JSONObject jsonObject) {
+		Pagination pagination = Pagination.of(
+			jsonObject.getInt("page"), jsonObject.getInt("pageSize"));
+
+		JSONArray itemsJSONArray = jsonObject.getJSONArray("items");
+
+		int length = itemsJSONArray.length();
+
+		List<SearchResult> searchResults = new ArrayList<>();
+
+		for (int index = 0; index < length; index++) {
+			searchResults.add(
+				SearchResult.toDTO(
```

Comment:
Is sourceformatter really not complaining about chaining here? I'd be on the safe side and not chain `itemsJSONArray.get(index).toString()` ...